ghost/core/core/server/models/base/plugins/overrides.js

```diff
@@ -88,7 +88,10 @@ module.exports = function (Bookshelf) {
             // CASE: get JSON of previous attrs
             if (options.previous) {
                 const clonedModel = _.cloneDeep(this);
-                clonedModel.attributes = this._previousAttributes;
+                clonedModel.attributes = _.omitBy(
+                    this._previousAttributes,
+                    (value, key) => key.startsWith('_pivot_')
```

Comment:
Would be best to use the constant straight from the bookshelf instead of introducing a magic string here - https://github.com/bookshelf/bookshelf/blob/7704fbc5e83a35357d89458d9269a7e739964296/lib/constants.js#L1C1-L1C1. If it's not exposed, so be it 😆  - leaving a comment for thwere the matic string comes from would help with future maintenance 👍

Reply:
Its not exposed directly, but is accessible. What do you think of [this](https://github.com/TryGhost/Ghost/pull/17142/commits/3258050aef87b5e86348137057917a0bdad1c014)? Although we are accessing something that is technically not exported, we still have the safety of the tests failing if something changes around this so could be a good trade-off over us having to introduce the magic string ourselves

Reply:
That's a perfect solution imo 👌